### PR TITLE
Fix classpaths in dottydoc tests

### PR DESCRIPTION
**Pull request by felixmulder**
In reference to #1934

### CODE REVIEW

**Comment by liufengyun** [comment]
It seems the test hangs, as well as other ci run today.

**Comment by felixmulder** [comment]
Seems like this passed our CI now? The `validate-main` job doesn't really matter, only the CLA part.

File: doc-tool/test/MarkdownTests.scala [hunk 1]
```diff
@@ -6,6 +6,7 @@ import org.junit.Assert._
 
 import dotc.core.Contexts.{ Context, ContextBase, FreshContext }
 import dotc.core.Comments.{ ContextDoc, ContextDocstrings }
+import java.io.{ File => JFile }
```
Comment (liufengyun) [inline doc-tool/test/MarkdownTests.scala]: It seems this import is not used.

**Comment by liufengyun** [comment]
Otherwise, LGTM

**Pull request merged**